checkout.html

```diff
@@ -0,0 +1,174 @@
+            	<option value="OR">Oregon</option>
+            	<option value="PA">Pennsylvania</option>
+            	<option value="RI">Rhode Island</option>
+            	<option value="SC">South Carolina</option>
+            	<option value="SD">South Dakota</option>
+            	<option value="TN">Tennessee</option>
+            	<option value="TX">Texas</option>
+            	<option value="UT">Utah</option>
+            	<option value="VT">Vermont</option>
+            	<option value="VA">Virginia</option>
+            	<option value="WA">Washington</option>
+            	<option value="WV">West Virginia</option>
+            	<option value="WI">Wisconsin</option>
+            	<option value="WY">Wyoming</option>
+            </select>
+          </label>
+
+          <label class="contact-csz">Zip Code
+            <input type="text" pattern="[0-9]{5}" name="customerZip" value="">
+          </label>
+        </fieldset>
+
+        <fieldset>
+          <legend>First Time Adopter?</legend>
+            <label>Yes
+              <input type="radio" name="firstTime" value="yes">
+            </label>
+            <label>No
+              <input type="radio" name="firstTime" value="no">
+            </label>
+        </fieldset>
+
+        <fieldset>
+          <legend>Pickup Location</legend>
+            <select class="" name="pickupLocation">
+              <option value="">Select</option>
+              <option value="Ballard">Ballard</option>
+              <option value="CapHill">CapHill</option>
+              <option value="Downtown">Downtown</option>
+              <option value="GreenLake">Green Lake</option>
+            </select>
+        </fieldset>
+
+        <section>
+          <h3>Dogs Adopted</h3>
+
+          <div class="cart-dog">
+            <h5>Lucas</h5>
+              <p>Cost to Adopt: $123.45</p>
+            <img src="images/lucas-checkout.jpg" alt="a picture of a black outside looking up" width="75">
+          </div>
+
+          <div class="cart-dog">
+            <h5>Duffy</h5>
+              <p>Cost to Adopt: $123.45</p>
+            <img src="images/duffy-checkout.jpg" alt="a picture of a dog outside looking up" width="75">
+          </div>
+
+        </section>
+
+        <section>
+          <h3>Total Adoption Cost</h3>
+            <p>$1234.56</p>
+        </section>
+
+        <input type="submit" name="submit" value="Submit">
+
+      </form>
+
+
+
+
+    </section>
+
+
+    <footer style="background-color:#ADD8E6;">
+      <img src="images/adoptadog-logo.svg" alt="white adopt a dog logo"  width="503" height="136">
+        <p>Aborpos nisto inci tet aut que volupti beruptiberro to ex esseque invenienihil imaximu scideru ptistiusam consequunt</p>
+        <nav>
+          <a href="index.html">Home</a> /
+          <a href="dogs.html">Dogs</a> /
+          <a href="blog.html">Blog</a>
+        </nav>
+      <p>© 2018 Adoptadog, LLC | Seattle, WA</p>
+    </footer>
+
+  </body>
+</html>
```

Comment:
You'll want to set your editor to add a blank line at the end of all files. It's nice to get into that habit now, for when you make more complicated files.